StudentsManager.Spa/src/index.css

```
@@ -12283,3 +12283,197 @@ details.soge-result-card summary.soge-result-history-toggle:focus-visible {
+    background: rgba(0, 0, 0, 0.5);
+    color: white;
+    padding: 6px 10px;
+    border-radius: 6px;
+    font-size: 12px;
+    white-space: nowrap;
+    z-index: 10;
+    pointer-events: none;
+    animation: fadeIn 0.15s ease-in-out;
+} */
+
+@keyframes fadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)